src/main.rs

```diff
@@ -72,6 +74,10 @@ fn main() {
                     eprintln!("Consider building with the `native-tls` feature enabled.");
                 }
             }
+            if native_tls && msg == "invalid minimum TLS version for backend" {
+                eprintln!();
+                eprintln!("Try running without the --native-tls flag.");
+            }
```

Comment:
It happens if you combine `--ssl tls1.3` with `--native-tls`. Reqwest creates the error because native-tls doesn't let you specify it. I'll add a test.